### PR TITLE
owo

### PR DESCRIPTION
**Pull request by moonheart08**
## About the PR 

**Screenshots**
Screenshot from 2021-11-25 01-20-09

:cl: moony
- add: Administrators have had an essential command added to their toolset.

### CODE REVIEW

**Review by ShadowCommander — APPROVED** [review]

**Review by Zumorica — COMMENTED** [review]
I'm not a fan of static methods

**Comment by PaulRitter** [comment]
> I'm not a fan of static methods

gotta be a bit more constructive like that, seems kinda the best way in this case afaict. how'd you propose to change it?

**Comment by Zumorica** [comment]
> > I'm not a fan of static methods
> 
> gotta be a bit more constructive like that, seems kinda the best way in this case afaict. how'd you propose to change it?

Literally just `EntitySystem.Get<OwOAccentSystem>().Accentuate(yourTextHere)`, no need for static

**Review by PJB3005 — COMMENTED** [review]
Pro-static gang

**Comment by PaulRitter** [comment]
> > > I'm not a fan of static methods
> > 
> > 
> > gotta be a bit more constructive like that, seems kinda the best way in this case afaict. how'd you propose to change it?
> 
> Literally just `EntitySystem.Get<OwOAccentSystem>().Accentuate(yourTextHere)`, no need for static

image

yeah no sorry im with the pro-static gang on this one.

if there might be a future possiblity for which you'd want this in a system, say because you may want to introduce some way to globally change behaviour over cvars or smth i'd understand it, but this is literally just a very simple text formatting algo, i dont see this being changed ever.

**Comment by metalgearsloth** [comment]
If you're gonna static at least get rid of the field with the dependency.

**Comment by PaulRitter** [comment]
> If you're gonna static at least get rid of the field with the dependency.

that field still gets used tho

**Comment by metalgearsloth** [comment]
If it weren't a static it would also get used in both methods

File: Content.Server/Speech/EntitySystems/OwOAccentSystem.cs [hunk 1]
```diff
@@ -19,28 +19,26 @@ public class OwOAccentSystem : EntitySystem
...
-            return message.Replace("!", RandomFace)
+            return message.Replace("!", random.Pick(Faces))
                 .Replace("r", "w").Replace("R", "W")
                 .Replace("l", "w").Replace("L", "W");
         }
 
         private void OnAccent(EntityUid uid, OwOAccentComponent component, AccentGetEvent args)
         {
-            args.Message = Accentuate(args.Message);
+            args.Message = Accentuate(args.Message, _random);
```
Comment (PaulRitter) [inline Content.Server/Speech/EntitySystems/OwOAccentSystem.cs:41]: @metalgearsloth it still does. what?

**Comment by PaulRitter** [comment]
for the component, there is literally 0 change in how much it uses the dependency

File: Content.Server/Speech/EntitySystems/OwOAccentSystem.cs (hunk 1 above)
Comment (metalgearsloth) [reply Content.Server/Speech/EntitySystems/OwOAccentSystem.cs:41]: It gets used 50% less.

**Review by PaulRitter — CHANGES_REQUESTED** [review]
im wrong do what the funny woman said

EntitySystem.Get<OwOAccentSystem>().Accentuate(yourTextHere)

**Comment by PaulRitter** [comment]
reposting @Clyybber's meme for preservation
image

**Review by ShadowCommander — APPROVED** [review]

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**